components/camel-rest-swagger/src/main/java/org/apache/camel/component/rest/swagger/RestSwaggerEndpoint.java

```diff
@@ -651,4 +705,18 @@ static String queryParameterExpression(final Parameter parameter) {
         return expression.toString();
     }
 
+    protected static HttpRequestBase createMethod(String url) {
+
+        // create http holder objects for the request
+        HttpMethods methodToUse = HttpMethods.GET;
+
+        HttpEntity requestEntity = new StringEntity((String) "", "");
```

Comment:
We don't need to cast to string here

Reply:
This code will be removed in favor of 1-line expression